examples/fluids/README.md

```diff
@@ -7,390 +7,499 @@ The state variables are mass density, momentum density, and energy density.
+  - `J`
+:::
+
+An example of the `rotation` mode can be run with:
+
+```
+./navierstokes -problem advection2d -wind_type rotation -implicit -stab supg
+```
+
+and the `translation` mode with:
+
+```
+./navierstokes -problem advection2d -wind_type translation -wind_translation 1,-.5
+```
+
+### Runtime options for Advection 3D (-problem advection)
+
+:::{list-table} Advection3D Runtime Options
+:header-rows: 1
+
+* - Option
+  - Description
+  - Default value
+  - Unit
+
+* - `-lx`
+  - Length scale in x direction
+  - `8000`
+  - `m`
+
+* - `-ly`
+  - Length scale in y direction
+  - `8000`
+  - `m`
+
+* - `-lz`
+  - Length scale in z direction
+  - `4000`
+  - `m`
+
+* - `-rc`
+  - Characteristic radius of thermal bubble
+  - `1000`
+  - `m`
+
+* - `-units_meter`
+  - 1 meter in scaled length units
+  - `1E-2`
+  -
+
+* - `-units_second`
+  - 1 second in scaled time units
+  - `1E-2`
+  -
+
+* - `-units_kilogram`
+  - 1 kilogram in scaled mass units
+  - `1E-6`
+  -
+
+* - `-strong_form`
+  - Strong (1) or weak/integrated by parts (0) residual
+  - `0`
+  -
+
+* - `-stab`
+  - Stabilization method (`none`, `su`, or `supg`)
+  - `none`
+  -
+
+* - `-CtauS`
+  - Scale coefficient for stabilization tau (nondimensional)
+  - `0`
+  -
+
+* - `-wind_type`
+  - Wind type in Advection (`rotation` or `translation`)
+  - `rotation`
+  -
+
+* - `-wind_translation`
+  - Constant wind vector when `-wind_type translation`
+  - `1,0,0`
+  -
+
+* - `-E_wind`
+  - Total energy of inflow wind when `-wind_type translation`
+  - `1E6`
+  - `J`
+
+* - `-bubble_type`
+  - `sphere` (3D) or `cylinder` (2D)
+  - `shpere`
+  -
+
+* - `-bubble_continuity`
+  - `smooth`, `back_sharp`, or `thick`
+  - `smooth`
+  -
+:::
+
+An example of the `rotation` mode can be run with:
+
+```
+./navierstokes -problem advection -wind_type rotation -implicit -stab supg
+```
+
+and the `translation` mode with:
+
+```
+./navierstokes -problem advection -wind_type translation -wind_translation .5,-1,0
+```
+
+### Runtime options for Euler Traveling Vortex (-problem euler_vortex)
+
+:::{list-table} Euler Vortex Runtime Options
+:header-rows: 1
+
+* - Option
+  - Description
+  - Default value
+  - Unit
+
+* - `-lx`
+  - Length scale in x direction
+  - `1000`
+  - `m`
+
+* - `-ly`
+  - Length scale in y direction
+  - `1000`
+  - `m`
+
+* - `-lz`
+  - Length scale in z direction
+  - `1`
+  - `m`
+
+* - `-center`
+  - Location of vortex center
+  - `(lx,ly,lz)/2`
+  - `(m,m,m)`
+
+* - `-units_meter`
+  - 1 meter in scaled length units
+  - `1E-2`
+  -
+
+* - `-units_second`
+  - 1 second in scaled time units
+  - `1E-2`
+  -
+
+* - `-mean_velocity`
+  - Background velocity vector
+  - `(1,1,0)`
+  -
+
+* - `-vortex_strength`
+  - Strength of vortex < 10
+  - `5`
+  -
+
+* - `-euler_test`
+  - Euler test option (`t1`-`t4` and `none`)
+  - `none`
+  -
+:::
+
+This problem represents the exact solution to five test cases, which are selected by `-euler_test`, where
+- `none`: Isentropic vortex
```

Comment:
perhaps we should make this `t0`? or `test_i` for all of the options?

Reply:
Done!